primitives/test-utils/src/lib.rs

```diff
@@ -31,29 +31,58 @@ use sp_std::prelude::*;
 pub const TEST_GRANDPA_ROUND: u64 = 1;
 pub const TEST_GRANDPA_SET_ID: SetId = 1;
 
-/// Get a valid Grandpa justification for a header given a Grandpa round, authority set ID, and
-/// authority list.
+/// Make a valid GRANDPA justification with sensible defaults
+pub fn make_default_justification<H: HeaderT>(header: &H) -> GrandpaJustification<H> {
+	make_justification_for_header(header, TEST_GRANDPA_ROUND, TEST_GRANDPA_SET_ID, &keyring(), 2, 1)
+}
+
+/// Generate justifications in a way where we are able to tune the number of pre-commits
+/// and vote ancestries which are included in the justification.
+///
+/// This is useful for benchmarkings where we want to generate valid justifications with
+/// a specific number of pre-commits (tuned with the "forks" parameter) and/or a specific
+/// number of vote ancestries (tuned with the "depth" parameter).
+///
+/// Note: This needs at least three authorities or else the verifier will complain about
+/// being given an invalid commit.
 pub fn make_justification_for_header<H: HeaderT>(
 	header: &H,
 	round: u64,
 	set_id: SetId,
 	authorities: &[(Keyring, AuthorityWeight)],
+	depth: u32,
+	forks: u32,
```

Comment:
There is already 6 parameters and 3 of them are plain numbers, I think it deserves a parameters structs now. Understanding what 2 and 1 in `&keyring(), 2, 1)` is supposed to mean will quickly fade away :)